app/app.py

```diff
@@ -35,52 +51,42 @@ def fix_transfer_encoding():
     """
 
     transfer_encoding = request.headers.get("Transfer-Encoding", None)
-    if transfer_encoding == u"chunked":
+    if transfer_encoding == "chunked":
         request.environ["wsgi.input_terminated"] = True
 
 
-@app.route('/')
-def hello_world():
-   return "deploy.altlab.dev is running!"
-
-
-@app.route('/<app_name>', methods=["POST"])
-def secret_assert(app_name):
-    if '..' in app_name or '/' in app_name:
-        return abort(401)
+@app.route("/<app_name>", methods=["POST"])
+def deploy_app(app_name: str):
+    # Attempt to prevent a path traversal attack:
+    # (See: https://owasp.org/www-community/attacks/Path_Traversal)
+    if ".." in app_name or "/" in app_name:
+        return abort(HTTPStatus.BAD_REQUEST)
 
     body = request.get_json()
     if not isinstance(body, dict):
-        return abort(401)
+        return abort(HTTPStatus.BAD_REQUEST)
```

Comment:
I decided to use descript HTTP statuses rather than 401 for everything so it's easier to debug when something goes wrong. The app source code is public knowledge, so there's no sense in trying to hide what the app is doing.

Reply:
I used 401 because I did not know about `HTTPStatus`. Thanks!